omod/src/main/resources/webModuleApplicationContext.xml

```diff
@@ -129,11 +139,39 @@
 				
 				<prop key="admin/provider/providerAttributeType.list">providerAttributeTypeList</prop>
 				<prop key="admin/provider/providerAttributeType.form">providerAttributeTypeForm</prop>	
-				
+
  			</props>
 		</property>
 	</bean>	
 
+ 	<!-- index pages - begin -->  
+	
+	<!-- admin -->
+	<mvc:view-controller view-name="module/legacyui/admin/index" path="admin/index.htm"/>				
```

Comment:
@jdegraft I re-read both comments and I see what the problem is now. As @wluyima suggested, /admin is not even mapped to the Spring 'openmrs' DispatcherServlet in web.xml. 

However it is possible to map /admin to the DispatcherServlet (I did a quick test and it works). @wluyima why do you think the mapping would need to be removed if the legacyui module is stopped, any more than any other mapping like **/*.htm or **/*.form would need to be removed? If you stop the legacyui module those URLs will all just go back to looking for JSPs that don't exist, but that's ok because those URLs aren't supposed to work without legacyui?